kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/commands/KieServerControllerDescriptorCommand.java

```diff
@@ -54,6 +54,7 @@
             @XmlElement(name = "release-id", type = ReleaseId.class),
             @XmlElement(name = "capability", type = Capability.class),
             @XmlElement(name = "arguments", type = String.class),
+            @XmlElement(name = "long", type = Long.class)
```

Comment:
why this is needed?

Reply:
it is needed for the marshaller to know the type for deserialized any long value, similar to what is done in [here](https://github.com/kiegroup/droolsjbpm-integration/blob/807af498aa79bd05855eefda29b08cde103c3909/kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/ServiceResponse.java#L59)

It fixes to following error for starting the scanner:
Caused by: com.fasterxml.jackson.databind.exc.InvalidTypeIdException: Could not resolve type id 'Long' into a subtype of [simple type, class java.lang.Object]: known type ids = [Date, JaxbByteArray, Object, arguments, capability, container-config, container-spec-details, container-spec-key, java.util.Date, org.kie.server.api.model.type.JaxbByteArray, process-config, release-id, rule-config, server-config, server-instance-key, server-template-details]

Reply:
sounds bit weird but so be it..

Reply:
Agree, I was expecting that the jackson marshaller would handle basic types.